scriptmodules/supplementary/audiosettings.sh

```diff
@@ -251,16 +251,21 @@ function _pulseaudio_audiosettings() {
         printMsgs "dialog" "PulseAudio is present, but not running.\nAudio settings cannot be set right now."
         return
     fi
-    while read sink_index sink_label; do
+    declare -a sinks
```

Comment:
We don't this syntax mostly -  `local sinks=()` is better imho (unless I'm missing something)

Reply:
Right, changed it and moved it up.